keps/sig-cloud-provider/1959-service-lb-class-field/kep.yaml

```diff
@@ -24,7 +24,7 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.21"
+# latest-milestone: "v1.21"
```

Comment:
@andrewsykim FYI, PRR approval will need to be part of the PR that sets this back to 1.21.